### PR TITLE
Time and tagLink test modifications

### PR DESCRIPTION
**Pull request by Phanx091**

### CODE REVIEW

File: src/atoms/TagLink/tests/TagLink.test.js [hunk 1]
```diff
@@ -5,28 +5,26 @@ import TagLink from '../TagLink';
...
+  expect(node).toBeDefined();
+  expect(node.getAttribute('href')).toBe('/the/url/path');
+  expect(node.innerHTML).toBe('Live from Here');
+});
+test('Throws an error when a isRequired value is missing', () => {
+  expect(() => {
+    render(<TagLink />);
+  }).toThrow();
+});
+test("We'll have to overhaul TagLink later anyway", () => {
```
Comment (kimthompson) [inline src/atoms/TagLink/tests/TagLink.test.js]: this test should probably be deleted. It's kind of there as a note, but wasn't followed up on, and is not really testing anything.

File: src/atoms/TagLink/tests/TagLink.test.js [hunk 2]
```diff
@@ -5,28 +5,26 @@ import TagLink from '../TagLink';
...
+  expect(true).toBe(true);
 });
```
Comment (kimthompson) [inline src/atoms/TagLink/tests/TagLink.test.js]: There's room here to test that the `elementClass` prop works as well.

File: src/atoms/TagLink/tests/TagLink.test.js [hunk 3]
```diff
@@ -5,28 +5,26 @@ import TagLink from '../TagLink';
...
-  });
-  test('Throws an error when a isRequired value is missing', () => {
-    expect(() => {
-      render(<TagLink />);
-    }).toThrow();
-  });
-  test("We'll have to overhaul TagLink later anyway", () => {
-    expect(true).toBe(true);
-  });
+test('TagLink props value matches as expected', () => {
```
Comment (kimthompson) [inline src/atoms/TagLink/tests/TagLink.test.js]: This is kind of a bad name that's held over from the original test. It's hard to tell what's being tested here from that. In this case, I'd probably say that renaming this to "can set tagName and href via props" would be a lot clearer.

File: src/atoms/TagLink/tests/TagLink.test.js [hunk 4]
```diff
@@ -5,28 +5,26 @@ import TagLink from '../TagLink';
 // automatically unmount and cleanup DOM after the test is finished.
```
Comment (kimthompson) [inline src/atoms/TagLink/tests/TagLink.test.js]: You should rename your `tests` folder to `__tests__`, as discussed.

**Review by kimthompson — CHANGES_REQUESTED** [review]

**Review by Phanx091 — COMMENTED** [review]
update

**Review by kimthompson — APPROVED** [review]

**Pull request merged**